### PR TITLE
Enable the default DRM

### PR DESCRIPTION
**Pull request by walterx-yan**
CTS reqiures feature DRM, so enable the default one.

Change-Id: I801e647ae8b93a07e35ffbe5772e5ae1706c5511
Tracked-On: OAM-84905
Signed-off-by: Yan, WalterX <walterx.yan@intel.com>

### CODE REVIEW

**Comment by walterx-yan** [comment]
@hongcheng-xie to review

File: groups/default-drm/doc.spec [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+--- deps
+
+    - sepolicy
+
+==== Options
+
+--- true
+this option will only enable default drm, when not explicitly selected in mixin spec file, the default option will be used.
+   --- code dir
+       - device/intel/mixins/groups/widevine
```
Comment (youhuazhu) [inline groups/default-drm/doc.spec]: we can remove this line, it's widevie dir.

**Comment by walterx-yan** [comment]
@lizhuangzhi please help +2 for this PR

**Comment by walterx-yan** [comment]
@chenlin3 please help review this. Thanks.

**Pull request merged**